packages/generator/templates/app/src/app/auth/components/LoginForm.tsx

```diff
@@ -0,0 +1,53 @@
+import {AuthenticationError, PromiseReturnType} from "blitz"
```

Comment:
```suggestion
'use client'

import {AuthenticationError, PromiseReturnType} from "blitz"
```

Just from reading the PR, I think all files need a use client, especially since this one uses a mutation. Am I missing something?

Reply:
@tordans thanks! yes this is missed from the file I will add it to be more specific. Since the page using this was marked as a client component, there will be no error using this as is, but for convention, It would be better to add your suggession.